libbeat/beat/beat.go

```diff
@@ -120,18 +127,29 @@ func Run(name string, version string, bt Beater) {
 	}
 }
 
+// Starts the beat
```

Comment:
This method doc doesn't start with the method name. Here's my suggestion.

```
// Start starts the Beat by parsing and interpreting the command line flags,
// loading and parsing the configuration file, and running the Beat. This
// method blocks until the Beat exits. If an error occurs will initializing
// or running the Beat it will be returned.
```